testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -465,6 +465,8 @@ def test_average_structure_in_memory(self, universe):
 
 class TestAlignmentProcessing(object):
     seq = FASTA
+    ref = FASTA_ref
```

Comment:
Calling these FASTA_ref and FASTA_tgt is somewhat confusing for future devs given it's mainly for a single test. Ideally I'd name them something a bit more explicit like "FASTA_NOGAP_ref".